### PR TITLE
Infrastructure metric sets

### PR DESCRIPTION
**Pull request by olaaun**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/MetricsConsumer.java [hunk 1]
```diff
@@ -1,21 +1,21 @@
 // Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.model.admin.monitoring;
 
+import ai.vespa.metrics.set.Metric;
+import ai.vespa.metrics.set.MetricSet;
```
Comment (yngveaasheim) [inline config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/MetricsConsumer.java:5]: Needed?

File: container-core/src/main/java/com/yahoo/container/jdisc/state/MetricsPacketsHandler.java [hunk 2]
```diff
@@ -208,11 +243,57 @@ private void addMetrics(MetricSet metricSet, ObjectNode packet) {
...
+            } else if (value instanceof GaugeMetric) {
+                GaugeMetric gauge = (GaugeMetric) value;
+                metrics.put(name + ".average", sanitizeDouble(gauge.getAverage()));
+                metrics.put(name + ".last", sanitizeDouble(gauge.getLast()));
+                metrics.put(name + ".max", sanitizeDouble(gauge.getMax()));
+                if (gauge.getPercentiles().isPresent()) {
+                    for (Tuple2<String, Double> prefixAndValue : gauge.getPercentiles().get()) {
+                        metrics.put(name + "." + prefixAndValue.first + "percentile", prefixAndValue.second.doubleValue());
+                    }
+                }
```
Comment (yngveaasheim) [inline container-core/src/main/java/com/yahoo/container/jdisc/state/MetricsPacketsHandler.java:262]: Should we include 'min', 'sum', 'count' suffixes here?

Comment (yngveaasheim) [reply container-core/src/main/java/com/yahoo/container/jdisc/state/MetricsPacketsHandler.java:262]: And 'rate'?

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/MetricsConsumer.java (hunk 1 above)
Comment (olaaun) [reply config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/MetricsConsumer.java:5]: Yes, used further down

**Pull request merged**